File: app/src/main/java/org/worshipsongs/domain/AuthorSong.java
package org.worshipsongs.domain;

import org.apache.commons.lang3.builder.ToStringBuilder;


/**
 * Created by Seenivasan on 3/24/2015.
 */
public class AuthorSong {

    private int authorId;
    private int songId;
    private Song song;
    private Author author;


    public int getAuthorId() {
        return authorId;
    }

    public void setAuthorId(int authorId) {
        this.authorId = authorId;
    }

    public int getSongId() {
        return songId;
    }

    public void setSongId(int songId) {
        this.songId = songId;
    }

    public Song getSong()
    {
        return song;
    }

    public void setSong(Song song)
    {
        this.song = song;
    }

    public Author getAuthor()
    {
        return author;
    }

    public void setAuthor(Author author)
    {
        this.author = author;
    }

    @Override
    public String toString()
    {
        ToStringBuilder stringBuilder = new ToStringBuilder(this);
        stringBuilder.append("authorname", getAuthor().getName());
        stringBuilder.append("song title", getSong().getTitle());
        return stringBuilder.toString();
    }
}

File: app/src/main/java/org/worshipsongs/domain/AuthorSong.kt
package org.worshipsongs.domain

import org.apache.commons.lang3.builder.ToStringBuilder


/**
 * Created by Seenivasan on 3/24/2015.
 */
class AuthorSong
{

    var authorId: Int = 0
    var songId: Int = 0
    var song: Song? = null
    var author: Author? = null

    override fun toString(): String
    {
        val stringBuilder = ToStringBuilder(this)
        stringBuilder.append("authorname", author!!.name)
        stringBuilder.append("song title", song!!.title)
        return stringBuilder.toString()
    }
}
